src/assets/accessories/finger_nails/finger_nails.asset.ts

```diff
@@ -0,0 +1,54 @@
+DefineAsset({
+	name: 'Artificial finger nails',
+	size: 'small',
+	graphics: 'graphics.json',
+	colorization: {
+		nails: {
+			name: 'Nails',
+			default: '#AC0909',
+		},
+	},
+	// size:150, y:550, X:0
+	preview: 'preview.png',
+	attributes: {
+		provides: [
+			'Accessory',
+		],
+		requires: [
+			'!Handcover',
+			'!Gloves',
```

Comment:
Might need to retest it afterwards, but after the cover attributes were added to all gloves/mittens, you can remove this specific no-gloves attribute, as the cover mechanic takes care of this case more generically.

Reply:
Yes, should do...